west.yml

```diff
@@ -99,7 +99,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 70994a63a43f8738acab62123b0e180505ee0443
+      revision: pull/393/head
```

Comment:
The PR in nrfxlib needs to be updated with libs built with latest PR against libmodem, just placing a NACK here to ensure it does not go in without it :)